docs/architecture/adr-023-gas-used-and-gas-price-estimation.md

```diff
@@ -70,7 +70,7 @@ func Median(gasPrices []float64) (float64, error) {
 }
 ```
 
-For the top/bottom 10%, they will be extracted after sorting the list of gas prices incrementally, and slicing `gasPrices[:len(gasPrices)*10/100]` and `gasPrices[len(gasPrices)*90/100:]` respectively.
```

Comment:
not sure how gasPrices is sorted, but drive by PRs are usually not welcome accepted here @Fallengirl 

please see existing issues or create an issue showing the problem